tests/integration-tests/tests/users/test_default_user_home.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import logging
+
+import pytest
+from assertpy import assert_that
+from remote_command_executor import RemoteCommandExecutor
+
+# flake8: noqa
+
+
+@pytest.mark.usefixtures("os", "scheduler", "instance")
+def test_default_user_local_home(
```

Comment:
Can we add this in an already existing test?

Reply:
I tried to find a test to put this in, but there wasn't a good category to add this in.